Error running tests

<!--- Provide a general summary of the issue in the Title above -->
In version 2.42.0, when running tests in our project (npm run t) an error occurs:

./node_modules/copc/lib/utils/getter.js:78:65-78 - Error: Module not found: Error: Can't resolve 'fs' in 'Angular\node_modules\copc\lib\utils'

copc is a package reference in itowns package.json. Copc has a file getter.ts (https://github.com/connormanning/copc.js/blob/master/src/utils/getter.ts) which references 'fs'. This package was removed by npm because it contained malicious code: https://www.npmjs.com/package/fs?activeTab=versions

--- a/package.json
+++ b/package.json
@@ -61,7 +61,7 @@
     "@mapbox/vector-tile": "^1.3.1",
     "@tmcw/togeojson": "^5.8.1",
     "@tweenjs/tween.js": "^18.6.4",
-    "copc": "^0.0.5",
+    "copc": "^0.0.6",
     "earcut": "^2.2.4",
     "js-priority-queue": "^0.1.5",
     "pbf": "^3.2.1",

--- a/webpack.config.cjs
+++ b/webpack.config.cjs
@@ -1,6 +1,5 @@
 const fs = require('fs');
 const path = require('path');
-const webpack = require('webpack');
 const ESLintPlugin = require('eslint-webpack-plugin');
 
 const mode = process.env.NODE_ENV;
@@ -81,12 +80,6 @@ module.exports = () => {
         plugins: [
             new ESLintPlugin({
                 files: include,
-            }),
-            // Prevent the generation of module fs for import on copc dependency
-            // See https://webpack.js.org/plugins/ignore-plugin/
-            new webpack.IgnorePlugin({
-                resourceRegExp: /^fs$/,
-                contextRegExp: /copc/,
             }),
         ],
         devServer: {

